vllm/v1/kv_cache_interface.py

```diff
@@ -87,6 +84,18 @@ class KVCacheTensor:
     size: int  # The size of KV cache Tensor in bytes
 
 
+@dataclass
+class VirtualLayer:
```

Comment:
Why do we call this `VirtualLayer`? I believe woosuk and you had some discussions among this. The issue of `VirtualLayer` is that you can't tell that it's related to KV cache and KV cache grouping. To me I feel like maybe some names like "GroupedLayerKV" will be more straightforward.

Reply:
Discussing in #hybrid-mem channel of slack.